h/resources.py

```diff
@@ -134,6 +134,38 @@ def __getitem__(self, pubid):
         return organization.logo
 
 
+class GroupResource(object):
```

Comment:
`GroupResource` and `OrganizationResource` here follow the pattern established by `AnnotationResource`—that is, the `Factory` functions actually query the DB, while `*Resource` functions take a model object and return a Resource.